src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,27 @@
+import { getUserById } from '../../services';
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUserById(todo.userId);
```

Comment:
Ensure that `getUserById` is correctly fetching the user data based on `todo.userId`. This function should be verified to return the correct user object or null if not found.